packages/ketcher-core/src/application/editor/modes/SequenceMode.ts

```diff
@@ -480,54 +480,65 @@ export class SequenceMode extends BaseMode {
   }
 
   private get keyboardEventHandlers() {
-    return {
-      delete: {
-        shortcut: ['Backspace', 'Delete'],
-        handler: () => {
-          const editor = CoreEditor.provideEditorInstance();
-          const previousNode = SequenceRenderer.previousNode;
-          const previousCaretPosition = SequenceRenderer.caretPosition;
-          const selections = SequenceRenderer.selections;
-          const modelChanges = new Command();
-          let nodesToDelete: NodesSelection;
-
-          if (selections.length) {
-            modelChanges.merge(this.deleteSelectedDrawingEntities());
-            nodesToDelete = selections;
-          } else if (previousNode) {
-            previousNode.monomers.forEach((monomer) => {
-              modelChanges.merge(
-                editor.drawingEntitiesManager.deleteMonomer(monomer),
-              );
-            });
-            nodesToDelete = [
-              [
-                {
-                  node: previousNode,
-                  nodeIndexOverall:
-                    SequenceRenderer.previousCaretPosition as number,
-                },
-              ],
-            ];
-          } else {
-            return;
-          }
+    const deleteNode = (direction: 'left' | 'right') => {
```

Comment:
I would propose to use enum instead of string literal type to modify it easier in future